web/views/metrics.erb

```diff
@@ -9,8 +9,7 @@
   </div>
 
   <div>
-    <form id="metrics-form" class="form-inline" action="<%= root_path %>filter/metrics" method="post">
-      <%= csrf_tag %>
```

Comment:
CSRF checks are skipped on GET requests anyway, so the tag is not needed.

Was there any explicit intention behind filtering only with POST?
Is there any risk (e.g. filter operations being super heavy on Redis) that I am missing?

Reply:
I don’t recall any specific reason. It’s likely just copied from the retry/schedule filtering.